Bio/Seq.py

```diff
@@ -2972,8 +3017,6 @@ def defined(self):
 
         Zero-length sequences are always considered to be defined.
         """
-        if len(self) == 0:
-            return True
         return False
```

Comment:
Why remove this? It now contradicts the docstring (and other classes) for zero-length sequences are considered to be defined?

Reply:
Because `len(self)` is never zero.

Reply:
I don't follow. I may need to check out the code and run it, but earlier the ``_UndefinedSequenceData`` method ``__bytes__`` explicitily considers the case ``self._length == 0``

Reply:
Good point. I have now removed the check for `self._length` in `__bytes__`.

Reply:
I suggest you also remove ``Zero-length sequences are always considered to be defined.`` from this docstring and other classes which cannot be zero length.